### PR TITLE
chore(readme): improve text

### PR DESCRIPTION
**Pull request by fforootd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,21 +9,48 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/caos/zitadel)](https://goreportcard.com/report/github.com/caos/zitadel)
 [![codecov](https://codecov.io/gh/caos/zitadel/branch/master/graph/badge.svg)](https://codecov.io/gh/caos/zitadel)
 
-> This project is in alpha state. The API will continue breaking until version 1.0.0 is released
+> This project is in a alpha state. The application will continue breaking until version 1.0.0 is released
 
 ## What Is It
 
-It will be an IAM ;-)
+`ZITADEL` is a Cloud Native Identity and Access Management solution. All serverside componentes are written in `Go` and the management interface, called `console`, is written in `Angular`.
```
Comment (juergrinaldi) [inline README.md:16]: components (not componentes)

File: README.md [hunk 2]
```diff
@@ -9,21 +9,48 @@
...
+
+We optimized ZITADEL for the usage as `service provider IAM`. By `service provider` we think of companies who build services for e.g SaaS cases. Often these companies would like to use a IAM where they can register there application and grant other people or companies the right to self manage a set of roles within that application.
```
Comment (juergrinaldi) [inline README.md:18]: an IAM (not a IAM)

File: README.md [hunk 3]
```diff
@@ -9,21 +9,48 @@
...
 
 ## How Does It Work
 
+We built `ZITADEL` around the idea that the IAM should be easy to deploy and scale. That's why we tried to reduce external systems as much as possible.
+For example, `ZITADEL` is eventsourced but it does not rely on a pub/sub system to function. We instead built all the functionality right into one binary.
```
Comment (juergrinaldi) [inline README.md:23]: Instead we built... (not: We instead built...)

File: README.md [hunk 4]
```diff
@@ -9,21 +9,48 @@
...
+`ZITADEL` only needs `kubernetes` for orchestration and `cockroachdb` as storage.
```
Comment (juergrinaldi) [inline README.md:24]: names usually uppercase (Kubernetes, CockroachDB)

File: README.md [hunk 5]
```diff
@@ -9,21 +9,48 @@
...
 ## Why Another IAM
 
+In the past we already built a closed sourced IAM and tested multiple others. With most of them we had some issues, either technology, feature, pricing or transparency related in nature. For example we find the idea that security related features like `MFA` should not be hidden behind a paywall or a feature price.
+One feature that we often missed, was a solid `audit trail` of all IAM resources. Most systems we saw so far either rely on simple log files or use a short retention for this.
+
 ## How To Use It
 
+### Use our free tier
+
+Stay tuned, we will soon publish how you can register yourself a organisation in our cloud offering `zitadel.ch`.
```
Comment (juergrinaldi) [inline README.md:35]: an organisation (not a organisation)

File: README.md [hunk 6]
```diff
@@ -9,21 +9,48 @@
...
+Yes we have a free tier!
+
+### Run your own IAM
+
+Stay tuned, we will publish soon a guide how you can deploy a `hyperconverged` system with our automation tooling called `ORBOS`.
```
Comment (juergrinaldi) [inline README.md:40]: not sure: Stay tuned, we will soon publish a guide.... (placement of soon)

File: README.md [hunk 7]
```diff
@@ -9,21 +9,48 @@
...
+With `ORBOS` you will be able to run `ZITADEL` on `GCE` or `StaticMachines` within 20min. To achieve this, `ORBOS` will boostrap a `kubernetes` cluster, install the platform components (logging, metrics, ingress, ...), start a secure `cockroach` cluster and run and operate the `ZITDADEL`.
```
Comment (juergrinaldi) [inline README.md:41]: 20 minutes (i would not abb.)

Comment (juergrinaldi) [inline README.md:41]: the Zitadel? I'd just write "...and operate `ZITADEL`.

File: README.md (hunk 2 above)
Comment (fforootd) [reply README.md:18]:
```suggestion
We optimized ZITADEL for the usage as `service provider IAM`. By `service provider` we think of companies who build services for e.g SaaS cases. Often these companies would like to use an IAM where they can register there application and grant other people or companies the right to self manage a set of roles within that application.
```

File: README.md (hunk 3 above)
Comment (fforootd) [reply README.md:23]:
```suggestion
For example, `ZITADEL` is eventsourced but it does not rely on a pub/sub system to function. Instead we built all the functionality right into one binary.
```

File: README.md (hunk 1 above)
Comment (juergrinaldi) [inline README.md:16]:
```suggestion
`ZITADEL` is a Cloud Native Identity and Access Management solution. All serverside components are written in `Go` and the management interface, called `console`, is written in `Angular`.
```

File: README.md (hunk 4 above)
Comment (fforootd) [reply README.md:24]:
```suggestion
`ZITADEL` only needs `Kubernetes` for orchestration and `Cockroachdb` as storage.
```

File: README.md (hunk 5 above)
Comment (fforootd) [reply README.md:35]:
```suggestion
Stay tuned, we will soon publish how you can register yourself an organisation in our cloud offering `zitadel.ch`.
```

File: README.md (hunk 2 above)
Comment (juergrinaldi) [inline README.md:18]:
```suggestion
We optimized ZITADEL for the usage as `service provider IAM`. By `service provider` we think of companies who build services for e.g SaaS cases. Often these companies would like to use an IAM where they can register there application and grant other people or companies the right to self manage a set of roles within that application.
```

File: README.md (hunk 6 above)
Comment (fforootd) [reply README.md:40]:
```suggestion
Stay tuned, we will soon publish a guide how you can deploy a `hyperconverged` system with our automation tooling called `ORBOS`.
```

File: README.md (hunk 7 above)
Comment (fforootd) [reply README.md:41]:
```suggestion
With `ORBOS` you will be able to run `ZITADEL` on `GCE` or `StaticMachines` within 20 minutes. To achieve this, `ORBOS` will boostrap a `kubernetes` cluster, install the platform components (logging, metrics, ingress, ...), start a secure `cockroach` cluster and run and operate the `ZITDADEL`.
```

File: README.md (hunk 3 above)
Comment (juergrinaldi) [inline README.md:23]:
```suggestion
For example, `ZITADEL` is eventsourced but it does not rely on a pub/sub system to function. Instead we built all the functionality right into one binary.
```

File: README.md (hunk 4 above)
Comment (juergrinaldi) [inline README.md:24]:
```suggestion
`ZITADEL` only needs `Kubernetes` for orchestration and `CockroachDB` as storage.
```

File: README.md (hunk 7 above)
Comment (fforootd) [reply README.md:41]:
```suggestion
With `ORBOS` you will be able to run `ZITADEL` on `GCE` or `StaticMachines` within 20min. To achieve this, `ORBOS` will bootstrap a `Kubernetes` cluster, install the platform components (logging, metrics, ingress, ...), start a secure `cockroach` cluster and run and operate `ZITDADEL`.
```

File: README.md (hunk 5 above)
Comment (juergrinaldi) [inline README.md:35]:
```suggestion
Stay tuned, we will soon publish how you can register yourself an organisation in our cloud offering `zitadel.ch`.
```

File: README.md (hunk 6 above)
Comment (juergrinaldi) [inline README.md:40]:
```suggestion
Stay tuned, we will soon publish a guide how you can deploy a `hyperconverged` system with our automation tooling called `ORBOS`.
```

File: README.md (hunk 7 above)
Comment (juergrinaldi) [inline README.md]:
```suggestion
With `ORBOS` you will be able to run `ZITADEL` on `GCE` or `StaticMachines` within 20 minutes. To achieve this, `ORBOS` will boostrap a `kubernetes` cluster, install the platform components (logging, metrics, ingress, ...), start a secure `cockroach` cluster and run and operate `ZITDADEL`.
```

File: README.md [hunk 8]
```diff
@@ -9,21 +9,48 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/caos/zitadel)](https://goreportcard.com/report/github.com/caos/zitadel)
 [![codecov](https://codecov.io/gh/caos/zitadel/branch/master/graph/badge.svg)](https://codecov.io/gh/caos/zitadel)
 
-> This project is in alpha state. The API will continue breaking until version 1.0.0 is released
+> This project is in a alpha state. The application will continue breaking until version 1.0.0 is released
 
 ## What Is It
 
-It will be an IAM ;-)
+`ZITADEL` is a Cloud Native Identity and Access Management solution. All serverside components are written in `Go` and the management interface, called `console`, is written in `Angular`.
```
Comment (fgerschwiler) [inline README.md:16]: should console also be uppercase?

File: README.md [hunk 9]
```diff
@@ -9,21 +9,48 @@
...
 ## How To Use It
 
+### Use our free tier
+
+Stay tuned, we will soon publish how you can register yourself a organisation in our cloud offering `zitadel.ch`.
+Yes we have a free tier!
+
+### Run your own IAM
+
+Stay tuned, we will publish soon a guide how you can deploy a `hyperconverged` system with our automation tooling called `ORBOS`.
```
Comment (fgerschwiler) [inline README.md:40]: it would be nice to link orbos repo

Comment (fforootd) [reply README.md:40]: Makes sense, will do this

File: README.md (hunk 8 above)
Comment (fforootd) [reply README.md:16]:
```suggestion
`ZITADEL` is a Cloud Native Identity and Access Management solution. All serverside components are written in `Go` and the management interface, called `Console`, is written in `Angular`.
```

**Pull request merged**